package.json

```diff
@@ -1,27 +1,29 @@
 {
   "name": "taskcluster-lib-config",
-  "version": "0.8.8",
+  "version": "0.9.0",
   "author": "Jonas Finnemann Jensen <jopsen@gmail.com>",
-  "description": "taskcluster-lib-config",
+  "description": "Configuration loader that injects env variables into YAML",
   "license": "MPL-2.0",
   "main": "./lib/config",
   "scripts": {
-    "compile": "rm -rf lib/ .test/ && babel -sd lib/ src/ && babel -sd .test/ test/",
+    "compile": "rm -rf lib/ && babel -sd lib/ src/",
```

Comment:
My reason for doing so was to ensure that we're testing the babel output with the raw node executable.  My concern is that the weird issues like `Array.prototype.includes` not being found could be corrected and fixed automatically by the register thing.  We should be able to verify that behaviour pretty easily, but I just want to make sure that we're testing the actual .js files that we're exporting and shipping.